includes/wikia/DefaultSettings.php

```diff
@@ -1676,6 +1676,11 @@
  */
 $wgDisableWAMOnHubs = false;
 
+/* @name wgTreatAsCorporate
+ * Treat a wiki as if it was a corporate wiki
+ */
+$wgTreatAsCorporate = false;
```

Comment:
Not exactly, we're adding this list to coporate pages list: https://github.com/Wikia/app/blob/CONCF-4/extensions/wikia/CorporateFooter/CorporateFooterController.class.php#L12